cmd/substreams/registry-login.go

```diff
@@ -3,11 +3,12 @@ package main
 import (
 	"errors"
 	"fmt"
-	"github.com/charmbracelet/lipgloss"
-	"github.com/streamingfast/cli/utils"
 	"os"
 	"path/filepath"
 
+	"github.com/charmbracelet/lipgloss"
+	"github.com/streamingfast/cli/utils"
+
 	"github.com/spf13/cobra"
```

Comment:
```suggestion
	"github.com/charmbracelet/lipgloss"
	"github.com/streamingfast/cli/utils"
    "github.com/spf13/cobra"
```

And a another round of goimports.

Reply:
Done